### PR TITLE
Allow to use cache in circleci.

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description
Use cache for test in circleci.

### CODE REVIEW

**Comment by felipemontoya** [comment]
Which cache are you refering to? the circleci cache for the requirements?

**Comment by andrey-canon** [comment]
> Which cache are you refering to? the circleci cache for the requirements?

yes

File: .circleci/config.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
       - run:
           name: activate virtualenv and install dependencies.
           command: |
-            python3 -m venv venv
+            test -d venv || virtualenv -p python3 ./venv
```
Comment (felipemontoya) [inline .circleci/config.yml:34]: what does `test -d venv` do?

Comment (andrey-canon) [reply .circleci/config.yml:34]: verify if there is a folder called venv

**Pull request merged**